examples/ipynb/webgl_example_1.ipynb

```diff
@@ -189,21 +191,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
```

Comment:
@QuLogic I opened it in the ipython notebook kernel for python2, which was why those changes took place. Should I revert that commit and redo it in the python3 kernel?